src/hashmap.js

```diff
@@ -19,8 +19,17 @@
      */
     prototype['get'] = function (key) {
         var hash = calculateHash(key);
-        var entry = this.entries[hash];
-        return entry ? entry.value : null;
+        var bucket = this.entries[hash];
+
+        if (bucket) {
+            for (var i = 0; i < bucket.length; i++) {
```

Comment:
@Dashed not sure if you said this before I did it, but already fixed in the master branch.